CHANGES.md

```diff
@@ -15,6 +15,8 @@
 
 - Add a code-action for syntactic and semantic movement shortcuts based on Merlin's Jump command (#1364)
 
+- Upstream the implementation of signature help to merlin-lib (#1375)
```

Comment:
CHANGES are for the users. Don't think they're going to find this information useful.